Formula/ice.rb

```diff
@@ -12,10 +12,13 @@ class Ice < Formula
   end
 
   option "with-java", "Build Ice for Java and the IceGrid Admin app"
+  option "without-php", "Build without Ice for PHP"
+  option "without-python", "Build without Ice for Python"
 
   depends_on "mcpp"
   depends_on :java => ["1.7+", :optional]
   depends_on :macos => :mavericks
+  depends_on :python => :recommended
```

Comment:
you should be able to change this to `depends_on :python => :recommended if MacOS.version <= :snow_leopard`